JS_PostClass/scriptPostClass.js

```diff
@@ -0,0 +1,365 @@
+class PostList {
```

Comment:
The same problem with creating brunch as in the previous PR. In this PR I should see only this file, not 27)

Reply:
fix, pleasr code style in this file